sssom/constants.py

```diff
@@ -139,6 +139,17 @@
     RDF_SEE_ALSO,
 ]
 
+PREDICATE_FLIP_DICTIONARY = {
+    "skos:closeMatch": "skos:closeMatch",
```

Comment:
Are there perhaps CONSTANT values somewhere we can use for this dict?

Reply:
Yes, same file..lol. Replaced